api/static/rest_framework_swagger/css/osf_screen.css

```diff
@@ -0,0 +1,1232 @@
+/* http://meyerweb.com/eric/tools/css/reset/ v2.0 | 20110126 */
+.swagger-ui-wrap html,
```

Comment:
Yes, but they are wrong. We have made changes to a few of the css files, and these are the ones that have changed. Somewhat better would be to have a place for us to put static files that we can reference for the modified template, and I would be interested in hearing suggestions for that. But we need mods to the css that will be used by rest_framework_swagger.


> On May 7, 2015, at 8:42 PM, Steven Loria <notifications@github.com> wrote:
> 
> In api/static/rest_framework_swagger/css/osf_screen.css:
> 
> > @@ -0,0 +1,1232 @@
> > +/* http://meyerweb.com/eric/tools/css/reset/ v2.0 | 20110126 */
> > +.swagger-ui-wrap html,
> The static files are bundled within the rest_framework_swagger distribution. Running manage.py collectstatic puts them in this static folder, but this is a build step. These assets are versioned by rest_framework_swagger
> 
> —
> Reply to this email directly or view it on GitHub.
>